src/lib/Mapper/PagerSearchContentToSearchDataMapper.php

```diff
@@ -20,7 +20,7 @@
 use eZ\Publish\Core\MVC\Symfony\Locale\UserLanguagePreferenceProviderInterface;
 use Pagerfanta\Pagerfanta;
 
-class PagerSearchContentToDataMapper
+class PagerSearchContentToSearchDataMapper implements PagerSearchDataMapper
```

Comment:
This is a BC break for this package. Any code which relied on `PagerSearchContentToDataMapper` is going to stop working. What you could do is to re-add this class in the form of `PagerSearchContentToDataMapper extends PagerSearchContentToSearchDataMapper`  and annotate it as `@deprecated since Ibexa v3.2, use PagerSearchContentToSearchDataMapper instead`